library/catalog/static/css/main.css

```diff
@@ -77,3 +77,143 @@
     margin-top: 90px;
     margin-bottom: 20px;
 }
+
+/*
+=====
+DEPENDENCES
+=====
+*/
+
+.screen-reader{
+    width: var(--screenReaderWidth, 1px) !important;
+    height: var(--screenReaderHeight, 1px) !important;
+    padding: var(--screenReaderPadding, 0) !important;
+    border: var(--screenReaderBorder, none) !important;
+
+    position: var(--screenReaderPosition, absolute) !important;
+    clip: var(--screenReaderClip, rect(1px, 1px, 1px, 1px)) !important;
+    overflow: var(--screenReaderOverflow, hidden) !important;
+}
+
+/*
+=====
+CORE STYLES
+=====
+*/
+
+.rating{
+    --uiRatingColor: var(--ratingColor, #eee);
+    --uiRatingColorActive: var(--ratingColorActive, #ffcc00);
+
+    display: var(--ratingDisplay, flex);
+    font-size: var(--ratingSize, 1rem);
+    color: var(--uiRatingColor);
+}
+
+.rating__control:nth-of-type(1):focus ~ .rating__item:nth-of-type(1)::before,
```

Comment:
Need more clarity for this, not sure why we need to have 5 nearly the same rules